doc/getting_started/getting_started.rst

```diff
@@ -231,13 +231,21 @@ without the Zephyr SDK`_ for details.
    and extract it on your file system. This example assumes the compiler was
    extracted to: :file:`<user folder>/gcc-arm-none-eabi-5_3-2016q1/`.
 
+   On macOS, the toolchain can also be installed using brew::
+
+      $ brew tap osx-cross/arm
```

Comment:
We're avoiding use of the prompt character in code samples, making it easier to cut/paste, so delete the ``$``